lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -5711,6 +5714,8 @@ SwiftASTContext::GetTypeInfo(void *type,
     swift_flags |= eTypeHasChildren | eTypeIsReference | eTypeHasValue;
     break;
   case swift::TypeKind::DynamicSelf:
+    swift_flags |= eTypeIsGeneric | eTypeIsBound | eTypeHasValue;
```

Comment:
is `HasValue` needed?

Reply:
Yes. Without it ValueObject will early-exit without resolving the dynamic type.

Reply:
Cheers. The magic of ValueObject'ing.